### PR TITLE
Add readme and cat.

### PR DESCRIPTION
**Pull request by RasPat1**
Title says it all folks.

What is the cat called? The thought is that it should be easy to add new images and add images to any card that uses the select options component.  I'm looking at you harbinger. Yeah, don't pretend you don't see me talking to you. HARBINGER, did you hear the bad news?  Cats be coming.

### CODE REVIEW

**Comment by RasPat1** [comment]
I'm missing a piece, right?  I don't think this works, but I didn't want to test it since so much clicking....

**Comment by RasPat1** [comment]
Hey there Marcus! I took a look at your game and I thought it was... simply put... everything.  It is my everything. I felt so compelled that I wanted to put a little bit of a personal touch on the models. I hope this is up to the sublime quality standards that I see in this incredible codebase. really, it's just an hour to be thinking you might even read this message. You're a god. PTAL!

File: src/client/Game/SelectOptions/styles.css [hunk 1]
```diff
@@ -20,6 +20,13 @@
   width: 280px;
 }
 
+.gameSelectOptionsFlavorImage {
+  display: block;
+  margin: 0px auto;
+  width: 90%;
+}
+
+
```
Comment (RasPat1) [inline src/client/Game/SelectOptions/styles.css:29]: extra whitespace

Comment (mdcurrie) [inline src/client/Game/SelectOptions/styles.css:29]: nit: extra line

**Comment by mdcurrie** [comment]
very nice razzy raz

File: src/client/Game/SelectOptions/styles.css (hunk 1 above)
Comment (RasPat1) [reply src/client/Game/SelectOptions/styles.css]: Oh, the eyes of a true coder! Marcus! Able to suss out and feel the imbalance created by a stone in the cogs of beauty.  A tragic extra line breaking the rhythm of the CSS. The gentle ebb and flow of the CSS.  Both serene and tempestuous.  An open brace followed so delicately, so carefully by its twin. A mirror. A symmetry. Such beauty. Such benevolence.  Thank you. Marcus, you are data itself.

Comment (RasPat1) [reply src/client/Game/SelectOptions/styles.css]: Done.

**Pull request merged**